Fb yahoo 286 trunk gd

aleksgor opened the pull request.

ikatsov commented on hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/lib/db/TestConstructQuery.java:
```diff
@@ -18,50 +18,82 @@
-  
+
+  /**
+   * test DBOutputFormat. Should works with DBRecordWriter.
```
typo - "Should work with..."

ikatsov commented on hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/db/TestDbClasses.java:
```diff
@@ -0,0 +1,180 @@
+  }
+
+  /**
+   * test DataDrivenDBInputFormat and his splitters.
```
typo - "and its splitters"

ikatsov commented on hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapred/lib/db/TestDBInputFormat.java:
```diff
@@ -0,0 +1,179 @@
+
+  /**
+   * 
+   * test DBRecordReader. This reader should creates keys, values, know about position.. 
```
typo - "should create keys..."

ikatsov commented on hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapred/lib/db/TestDBInputFormat.java:
```diff
@@ -0,0 +1,179 @@
+    reader.close();
+  }
+/** 
+ * test configuration for db. should works DBConfiguration.* parameters. 
```
typo - "should work with DBConfiguration"

dennisyv commented:
looks fine

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

Commit:
Fixes according to the feedback

Commit:
Merge branch 'fb-YAHOO-286-trunk-gd' of github.com:/griddynamics/yhadoop-common into fb-YAHOO-286-trunk-gd

Commit:
Remove unnecessary differences between branch-2 and trunk patches

Commit:
Set number of maps to 1 explicitely as in the mapred-default.xml it's 2

Commit:
Remove Override annotation for methods not existed in JDK 1.6

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

Commit:
Get rid of setAccessible usage

Commit:
Merge branch 'fb-YAHOO-286-trunk-gd' of github.com:/griddynamics/yhadoop-common into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

Commit:
Remove non needed comment

Commit:
Use multiple splits in TestSplitters

Commit:
Merge branch 'fb-YAHOO-286-trunk-gd' of github.com:/griddynamics/yhadoop-common into fb-YAHOO-286-trunk-gd

Commit:
Get rid of non visible characters in TestDbClasses

Commit:
Remove useless tests

Conflicts:
	hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/lib/db/TestConstructQuery.java

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd

yahoo-bot committed:
Merge remote branch 'origin/trunk-gd' into fb-YAHOO-286-trunk-gd
